subscriber_topic: /videofile/image_raw
publisher_topic: /line_detector/output_image
publisher_topic2: /line_detector/threshold_image
subs_queue_size: 1
pubs_queue_size: 1
buff_size: 52428800
default_sobel_kernel_size: 7

img_size: [540, 960]

# Values for Region of Interest (ROI) Cutter
left_bottom: [40, 540]
right_bottom: [920, 540]
apex1: [450, 310]
apex2: [500, 310]
inner_left_bottom: [140, 540]
inner_right_bottom: [840, 540]
inner_apex1: [450, 540]
inner_apex2: [500, 540]

color_config:
  dilatation_kernel_size: 3
  canny_threshold: [80, 200]
  hough_threshold: 20
  hough_min_line_length: 3
  hough_max_line_gap: 1
  sobel_threshold: 40

  hsv_white1:  [0,0,150]
  hsv_white2:  [180,50,255]
  hsv_yellow1: [25,120,90]
  hsv_yellow2: [45,255,255]
  hsv_red1:    [0,140,100]
  hsv_red2:    [15,255,255]
  hsv_red3:    [165,140,100]
  hsv_red4:    [180,255,255]
